### PR TITLE
Attempted to fix issue 113

### PR DESCRIPTION
**Pull request by shaobo-he**

### CODE REVIEW

File: lib/seadsa/DsaLocal.cc [hunk 1]
```diff
@@ -1697,16 +1697,12 @@ void BlockBuilderBase::visitCastIntToPtr(const Value &dest) {
...
-    if (!m_graph.isFlat()) {
-      llvm::errs() << "WARNING: ";
-      bool printAddress = true;
-      LOG("dsa", llvm::errs() << dest; printAddress = false);
-      if (printAddress) llvm::errs() << "inttoptr @ addr " << &dest;
-      llvm::errs()
-          << " is (unsoundly) assumed to point to a fresh memory region.\n";
-    }
-  }
+  LOG(
```
Comment (agurfinkel) [inline lib/seadsa/DsaLocal.cc:1700]: move log to the error message itself. not if statement.

CI seems to not like the PR

use dsa-warn for log level. We use dsa for lot of other things

**Review by agurfinkel — CHANGES_REQUESTED** [review]

File: lib/seadsa/DsaLocal.cc (hunk 1 above)
Comment (shaobo-he) [reply lib/seadsa/DsaLocal.cc]: Thank you for the feedback. It's been addressed.

**Review by agurfinkel — APPROVED** [review]
@caballa I think I saw some CI failure for this, but it should be dependent on the change.
Can't find the failure now. This looks good to me, but I want to check with you before rebasing.

**Comment by shaobo-he** [comment]
> @caballa I think I saw some CI failure for this, but it should be dependent on the change.
> Can't find the failure now. This looks good to me, but I want to check with you before rebasing.

Hi @agurfinkel, I just applied the changes to other similar warning messages.

**Pull request merged**